images/hooks/cmd/002-tls-certificates-dvcr/main.go

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright 2025 Flant JSC
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+	"hooks/pkg/common"
+
+	tlscertificate "github.com/deckhouse/module-sdk/common-hooks/tls-certificate"
+	"github.com/deckhouse/module-sdk/pkg"
+	"github.com/deckhouse/module-sdk/pkg/app"
+	"github.com/tidwall/gjson"
+)
+
+func dvcrGetServiceIP(input *pkg.HookInput) gjson.Result {
+	return input.Values.Get(fmt.Sprintf("%s.internal.dvcr.serviceIP", common.MODULE_NAME))
+}
+
+func dvcrSans(sans []string) tlscertificate.SANsGenerator {
```

Comment:
```suggestion
func dvcrSANs(sans []string) tlscertificate.SANsGenerator {
```

for consistency with other names